scripts/lisk/start.sh

```diff
@@ -4,18 +4,32 @@ command -v shellcheck > /dev/null && shellcheck "$0"
 
 # A temporary dir created to make the docker-compose files from
 # https://github.com/LiskHQ/lisk/tree/v1.4.0/docker available.
-# Only used as long as this script runs
-TMP_DIR=$(mktemp -d "${TMPDIR:-/tmp}/lisk_start.XXXXXXXXX")
+TMP_DIR="${TMPDIR:-/tmp}/@iov"
+mkdir "$TMP_DIR" || true
```

Comment:
When you use `mkdir -p`, the command does not fail when the directory already exists. This avoids error messages in this case.